cmd/mirror-main.go

```diff
@@ -232,8 +232,8 @@ func (mj *mirrorJob) doRemove(sURLs URLs) URLs {
 	contentCh := make(chan *clientContent, 1)
 	contentCh <- &clientContent{URL: *newClientURL(sURLs.TargetContent.URL.Path)}
 	close(contentCh)
-
-	errorCh := clnt.Remove(false, contentCh)
+	isRemoveBucket := true
```

Comment:
I think we should not remove the bucket in the target if the source has zero contents. This should be `false`.
 
![image](https://user-images.githubusercontent.com/5410427/52200848-d164f680-288f-11e9-8f5a-609b50bf9cec.png)

Fine if it is left intentionally to work like this.

Reply:
@Praveenrajmani , changed it so mirror will not remove buckets